benchflow-faban-client/src/main/java/cloud/benchflow/faban/client/FabanClient.java

```diff
@@ -177,7 +187,60 @@ public RunStatus status(RunId runId) throws FabanClientException, RunIdNotFoundE
   }
 
   /**
+   * Get the Run Info of a Faban benchmark.
+   *
+   * @param runId a run id
+   * @return a response enclosing the run Info of the operation
+   * @throws FabanClientException when there is an error interacting with faban
+   * @throws RunIdNotFoundException when the run id is not found
+   */
+  public RunInfo runInfo(RunId runId) throws FabanClientException, RunIdNotFoundException {
+
+    RunConfig runConfig = new RunConfig(runId);
+    RunInfoCommand runInfo = new RunInfoCommand().withConfig(runConfig);
+    FabanClientConfig fabanConfig = chooseConfig();
+
+    try {
+      return runInfo.exec(fabanConfig);
+    } catch (IOException e) {
+      throw new FabanClientException(
+          "Something went wrong while requesting run info with runId" + runId, e);
+    }
+
+  }
+
+  /**
+   * Get the Run Info of a Faban benchmark, handling the result with an handler.
```

Comment:
Foxed in https://github.com/benchflow/benchflow/pull/434/commits/9807ecc0846d663c23722b4e1fe3e2c496c26e1a